stdlib/private/StdlibUnittest/RaceTest.swift

```diff
@@ -562,7 +562,12 @@ class _InterruptibleSleep {
       return
     }
 
+    #if os(WASI)
+    // WebAssembly/WASI on wasm32 is the only 32-bit platform with Int64 time_t
+    var timeout = timeval(tv_sec: time_t(duration), tv_usec: 0)
```

Comment:
Is it not safe to always cast the `duration` to `time_t`?

Reply:
`duration` is `Int` here, which is 32-bit on wasm32, do you mean a force cast?

```swift
duration as? time_t
```

Would that be safe then?